src/components/AddBook.js

```diff
@@ -1,9 +1,42 @@
+import { useDispatch } from 'react-redux';
+import { useState } from 'react';
+import { addBook } from '../redux/books/booksSlice';
+
 export default function AddBook() {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBook = {
+      id: Math.random().toString().substr(2, 6),
+      title,
+      author,
```

Comment:
- [ ] Kindly make sure to change the property `id` to `item_id` because when removing the book in the redux store you have used the `item_id`  property to filter the books. Kindly make sure to change it because we can not be able to remove the newly added book when we remove one book all newly added books are deleted at the same time.

Reply:
Thank you dear, I'll fix that.